src/anbox/utils.cpp

```diff
@@ -193,5 +194,17 @@ bool is_mounted(const std::string &path) {
 
 }
 
+std::string find_program_on_path(const std::string &name) {
```

Comment:
could be find_full_path :) however given your rationale I will not argue, find_program_on_path is fine.